client/verta/verta/_deployment/endpoint.py

```diff
@@ -128,23 +129,21 @@ def _get_json_by_path(cls, conn, workspace, path):
                 return endpoint
         return None
 
-    def update(self, run, strategy, wait=False, resources=None, autoscaling=None, env_vars=None):
-        if not isinstance(run, experimentrun.ExperimentRun):
-            raise TypeError("run must be an ExperimentRun")
+    def update(self, model_reference, strategy, wait=False, resources=None, autoscaling=None, env_vars=None):
+        if not (isinstance(model_reference, experimentrun.ExperimentRun) or isinstance(model_reference, RegisteredModelVersion)):
```

Comment:
`isinstance()` supports a tuple as its second argument to check against multiple types
```suggestion
        if not isinstance(model_reference, (experimentrun.ExperimentRun, RegisteredModelVersion)):
```